src/Agent/NewRelic/Agent/Extensions/Providers/Wrapper/AwsSdk/AwsSdkPipelineWrapper.cs

```diff
@@ -38,6 +40,24 @@ private string GetRegion(IAgent agent, dynamic requestContext)
             return "";
         }
 
+        private string GetAccountId(IAgent agent)
+        {
+            if (_accountId != null)
+            {
+                return _accountId;
+            }
+            _accountId = agent.Configuration.AwsAccountId;
+            if (_accountId == null)
+            {
+                _accountId = "";
+            }
+            else if ((_accountId.Length != 12) || _accountId.Any(c => (c < '0') || (c > '9')))
+            {
+                agent.Logger.Warn("Supplied AWS Account Id appears to be invalid: {0}", _accountId);
```

Comment:
Do we need to do something here so that this codeql check does not always get triggered? Maybe we don't need to log the actual id, because it should be available in the settings payload?

Reply:
I opted for logging it instead of putting it in the reported settings but we don't have to do it that way. The account ID isn't sensitive info so I'd probably just dismiss the warning, but it's also not essential if we just want to skip logging it. We can talk it over in pairing.

Reply:
Yes, we could dismiss it, but every time this code gets touched, it would trigger yet another warning. So it would just be nice to avoid needing to dismiss it every time.